src/app/shared/filter/episodes.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs/Subscription';
+import { selectEpisodeFilter, selectEpisodes } from '../../ngrx/reducers';
+import { EpisodeModel } from '../../ngrx/model';
+import { CastleFilterAction } from '../../ngrx/actions';
+
+@Component({
+  selector: 'metrics-episodes',
+  template: `
+    <span>Episodes:</span>
+    <span>
+      <prx-select [options]="allEpisodeOptions" [selected]="selected" searchable="true" (onSelect)="onEpisodesChange($event)"></prx-select>
```

Comment:
Seems like there are some UX issues with how the select works here.  May need to address them in styleguide at some point.

No results rendered until you start typing, so it's hard to see/alter the selections.  In this case, would probably be best if it rendered the "most recent N", but typing in the filter can get you to the older episodes.  Not sure what to do about the "selected but not visible per the filter" problem.